reference/introduction_to_the_buildsystem.rst

```diff
@@ -86,6 +86,20 @@ Example for using 4 cores:
 
     user@host:~/godot$ scons platform=x11 -j 4
 
+The default build is not heavily optimized, to allow
```

Comment:
Actually not optimized at all, it's a debug build :)

Could you make a new section for that? You put it at the right place, but it shouldn't be under "Platform selection", best make a new "Optimized build" section.